src/pages/shift-left-the-good-parts.tsx

```diff
@@ -40,7 +39,7 @@ const ShiftLeftPage: FunctionComponent = () => (
         className="bg-white"
         hero={
             <div className="sg-bg-gradient-venus">
-                <div className="container py-6 lg:tw-py-5xl md:tw-text-center">
+                <div className="container pt-6 lg:tw-pt-5xl md:tw-text-center">
```

Comment:
Think this needs to be using `py`, and we can probably remove Bootstrap here too

```suggestion
                <div className="tw-max-w-screen-md tw-px-sm tw-mx-auto tw-py-5xl md:tw-text-center">
```

Or rather, can we use the `<Hero />` component here instead for consistency?

![Screen Shot 2022-09-23 at 12 16 19 AM](https://user-images.githubusercontent.com/1733936/191891122-652b1c8d-ff2a-4422-be18-0e986ed773a4.png)